sympy/calculus/tests/test_util.py

```diff
@@ -66,6 +66,10 @@ def test_function_range():
     raises(NotImplementedError, lambda : function_range(
         sin(x)/2, x, S.Naturals))
 
+@slow
+def test_function_range1():
+    assert function_range(tan(x)**2 + tan(3*x)**2 + 1, x, S.Reals) == Interval(1,oo)
+
 
 def test_continuous_domain():
```

Comment:
There should be two blank lines between top-level functions.

Reply:
I corrected that